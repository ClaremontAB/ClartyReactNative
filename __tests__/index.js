import 'react-native';
import React from 'react';
import ClartyReactNative from '../app/index';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  jest.mock('WebView', () => 'WebView')

  const tree = renderer.create(
    <ClartyReactNative />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
