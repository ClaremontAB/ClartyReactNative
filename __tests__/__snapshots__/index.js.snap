// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <WebView
    automaticallyAdjustContentInsets={false}
    decelerationRate="normal"
    domStorageEnabled={true}
    injectedJavaScript="document.getElementsByTagName('body')[0].style.height = '1334px';"
    javaScriptEnabled={true}
    javaScriptEnabledAndroid={true}
    onLoadEnd={[Function]}
    onNavigationStateChange={[Function]}
    onShouldStartLoadWithRequest={[Function]}
    scalesPageToFit={true}
    source={
      Object {
        "uri": "https://login.microsoftonline.com/common/oauth2/authorize?response_type=code&client_id=f4d22284-40ab-4d51-847d-f5bfb55b1446",
      }
    }
    startInLoadingState={false}
    style={
      Array [
        undefined,
        Object {
          "alignSelf": "stretch",
          "flex": 1,
          "height": 1334,
          "width": 750,
        },
      ]
    }
  />
</View>
`;
